File: Atv11/Retangulo.py
class Retangulo:

    def __init__(self, comprimento, largura):
        self.comprimento = comprimento
        self.largura = largura
    
    def calcular_area(self):
        return self.comprimento * self.largura
    
    def calcular_perimentro(self):
        return 2 * (self.comprimento + self.largura)
    
ret = Retangulo(200, 300)
    
print(f"Área: {ret.calcular_area()}m")
print(f"Perimentro: {ret.calcular_perimentro()}m")

File: Aula03/Atv13/Retangulo.java
package Atv13;

public class Retangulo {
    private double comprimento;
    private double largura;

    public Retangulo(double comprimento, double largura) {
        this.comprimento = comprimento;
        this.largura = largura;
    }

    public double calcularArea() {
        return comprimento * largura;
    }

    public double calcularPerimetro() {
        return 2 * (comprimento + largura);
    }

    public static void main(String[] args) {
        Retangulo ret = new Retangulo(200, 300);
        System.out.println("Área: " + ret.calcularArea());
        System.out.println("Perímetro: " + ret.calcularPerimetro());
    }
}